tsl/test/expected/continuous_aggs_ddl.out

```diff
@@ -940,5 +940,105 @@ DROP MATERIALIZED VIEW whatever_view_1;
 NOTICE:  drop cascades to table _timescaledb_internal._hyper_13_24_chunk
 DROP MATERIALIZED VIEW whatever_view_2;
 NOTICE:  drop cascades to table _timescaledb_internal._hyper_14_25_chunk
+-- test bucket width expressions
+CREATE TABLE metrics_int2 (
```

Comment:
FWIW, tests on all types of tables already exists in other tests, e.g., `continuous_aggs_refresh`. I suggest repurposing/extending those tests instead of adding new ones that are basically the same.